backend/internal/controller/server/msg_handler.go

```diff
@@ -143,14 +143,20 @@ func (s server) ManipulateScoreHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	switch action := r.URL.Query().Get(config.AttributeAction); action {
-	case "add":
-		err := s.game.AddGoal(teamID)
-		if err != nil {
+	case config.ActionAdd:
+
+		if err := s.game.UpdateManualGoals(teamID, config.ActionAdd); err != nil {
 			log.Error(err)
 		}
-	case "sub":
-		err := s.game.SubGoal(teamID)
-		if err != nil {
+		if err := s.game.AddGoal(teamID); err != nil {
+			log.Error(err)
```

Comment:
Can you extract this `log.Error(err)` outside of switch? This instruction is repeated many times.

Reply:
Not really. It wouldn't let us trace multiple errors that may occur here. For example if UpdateManualGoals() would return an error it could be overwritten by an error caused by AddGoal().
I don't have any other idea how to do it better but I think this solution is ok for us.

Reply:
Ok for me. My questions are aimed at reflecting on the sense of the solution - not necessarily changing the current solution.

Now I can see that you can use the error list here and process them after the `switch`, but do you really want to perform the next operation when you have the error from the previous one (maybe we don't need the list)?

Reply:
Good point. We discussed it and changed this piece of code.

Reply:
OK, thread to close (I do not have permission to close any thread)